### DISCUSSION TITLE
When will the C++ plugin be back?

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#4579] [General]
### Description

Like the title.

### Use case/motivation

Have been waiting too long. There used to be a roadmap of including it. Now that roadmap no longer available. The discussion of this repo is a ghost town. No one replied.

https://github.com/apache/netbeans/discussions/4574

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by ghost** [comment]
I have already posted a similar thread in Discussions. See https://github.com/apache/netbeans/discussions/4574. This place is a ghost town. No one replied. Now you moved my issue to this forum again. Very clear you want to shut me up. Good job.

**Reply by geertjanw** [reply]
Yes, in the other thread you refer to, nobody responded when you asked: " I want to know when will it's back. Exactly when."

There isn't a single open source project, consisting of people who collaborate without making any demands of each other, where anybody would respond to that.

**Reply by neilcsmith-net** [reply]
@va0dl914 I moved it but hadn't got around to responding.  Opening invalid issues is not the way to get a response.  It just creates noise that people have to spend time sorting out.

CND isn't currently in a releasable state, for a number of reasons / complications.  See discussions on dev@ and in pull requests, and feel free to jump in and help out.

**Reply by Chris2011** [reply]
@va0dl914 I really can understand your frustration and you will be heard. However if you followed the mailing lists, there was some discussion about months, probably years ago. And as far as I can remember and as @neilcsmith-net already said, it wasn't in a releasble state also due to licensing issues. So I believe, the old one will never come back as it was before and will not be part of the core again. Yes, the new one is less powerfull than the old one but it grows with you and everyone else. However and afaik, you still can use the old cnd module, by adding an old Catalog, but no guarantee that everything works. And keep in mind, please don't create issues, if you find one by using the old module, because it is not maintained anymore and will not be part of the core. I know there where a discussion how you can add the old cnd to NetBeans (latest 15) for example.

For more information about the old CND cluster, just have a look into this page: https://cwiki.apache.org/confluence/display/NETBEANS/4th+and+5th+donations
